vmdb/app/models/chargeback.rb

```diff
@@ -58,7 +58,6 @@ def self.build_results_for_report_chargeback(options)
     $log.info("#{log_prefix} Calculating chargeback costs...")
 
     tz = Metric::Helper.get_time_zone(options[:ext_options])
-    tp = options[:ext_options][:time_profile] #TODO: Support time profiles
```

Comment:
Can we leave the `TODO` comment in?  or maybe change the comment to `# TODO: Support time profiles via options[:ext_options][:time_profile]`